assets/react/controllers/Incident/TaskTimeline.jsx

```diff
@@ -0,0 +1,7 @@
+import React from "react";
+import TaskTimeline from "../../../../js/app/dashboard/components/TaskTimeline";
+
+export default function ({ events }) {
+  events = Object.values(JSON.parse(events));
```

Comment:
I would not modify props, just in case. Why not create a new variable?

Reply:
The initial props binding is unusable without parsing it soo idk :shrug: no strong opinion on this :)

Reply:
what I meant is just creating a separate variable instead of modifying the props, something like:

```
export default function ({ _events }) {
const events = Object.values(JSON.parse(_events));
``